datajunction-server/datajunction_server/models/node.py

```diff
@@ -636,7 +636,12 @@ class NodeRevision(NodeRevisionBase, table=True):  # type: ignore
         },
     )
 
-    parent_links: List[NodeRelationship] = Relationship()
+    parent_links: List[NodeRelationship] = Relationship(
+        sa_relationship_kwargs={
+            "primaryjoin": "NodeRevision.id==NodeRelationship.child_id",
+            "cascade": "all, delete",
```

Comment:
This lets us cascade the `NodeRevision`'s delete to the `NodeRelationship` table